### DISCUSSION TITLE
上传spine动画三个文件后报错

### DISCUSSION THREAD

**Discussion by JXD28** [galacean/discussion#116] [Editor] [answered]
![报错信息](https://img.alicdn.com/imgextra/i1/O1CN01IehFCL20YOJ7Ud4ob_!!6000000006861-0-tps-2052-1354.jpg)

素材：
[素材.zip](https://github.com/galacean/discussion/files/13439363/default.zip)

素材来源为：https://oasis.alipay.com/store -> spine Tab -> 随便找一个下载

**Comment by singlecoder** [comment, marked as answer]
素材文件名没对上
